src/app.js

```diff
@@ -1 +1,30 @@
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function copyFile() {
+  const terminalParameters = process.argv.slice(2);
+  const copyFromFile = terminalParameters[0];
+  const copyToFile = terminalParameters[1];
+  const pathToCopyFromFile = path.join(__dirname, copyFromFile);
```

Comment:
Check if both `copyFromFile` and `copyToFile` are defined by user before next steps.